web/react/components/post_list.jsx

```diff
@@ -83,6 +83,7 @@ export default class PostList extends React.Component {
         };
     }
     componentDidMount() {
+        window.onload = () => this.scrollToBottom();
```

Comment:
It is strange, I spent 3 hours trying to do it another way. iOS does this weird thing with React where it waits for everything to render before loading the DOM so this was the only way to scroll at the correct time on a new page load.

I can add a condition to only do it when it's iOS but then if it occurs on other devices we'd have to maintain the condition. Leaving it in for all devices didn't cause any issues I could find